### DISCUSSION TITLE
Failure to get benefits pf GitHub student dev pack.

### DISCUSSION THREAD

**Discussion by Sidharth-Malik0** [community/community#50126] [GitHub Education]
Everytime I apply for GitHub student dev pack, it keeps rejecting me saying "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Mar 15, 2023 for @Sidharth-Malik0". What could be the reason? I have Clearly included the date of enrolment and when it will expire and everything possibly needed, still its rejecting

P.S- I have tried 8 times.

**Comment by Ntshangase** [comment]
check out : https://github.com/community/community/discussions/17814

**Reply by Sidharth-Malik0** [reply]
I ve already done so. I meet all the criterions, still I am unable to get the benefits. I dont have a school email but my id card states my enrolment datee, age and other info

**Reply by Ntshangase** [reply]
Maybe @samus-aran can be able to help. 

What I would suggest you do is provide more information about  your application and why it was rejected that can help us better understand and be able to assist you.

GitHub has made changes to how they handle GitHub education support, I would suggest you check it out: https://github.com/orgs/community/discussions/49366

**Reply by jarvisa98** [reply]
The GitHub Student Developer Pack is a fantastic resource for students looking to kickstart their development journey by providing access to a wide range of tools and services to visit to the[ rope hero mod apk ]([https://ropeheroapk.pro/](https://ropeheroapk.pro/rope-hero-vice-town-1-3-1-mod-apk/))

**Comment by Sidharth-Malik0** [comment]
![Screenshot_20230316_170308](https://user-images.githubusercontent.com/127592068/225604802-93e6e0bf-3434-4e45-9f43-486c536d2aa9.jpg)
I uploaded this image. (I have hidden personal info while podting here, but I posted whole image without editing in the email.)

**Comment by Sidharth-Malik0** [comment]
![Screenshot_20230316_170145_com google android gm](https://user-images.githubusercontent.com/127592068/225604972-dffb581d-e8f2-4e12-aba6-fbdc303f99c9.jpg)
I get this response every time I re submit. I have submitted 8times

**Reply by Sidharth-Malik0** [reply]
I ve been trying from different accounts since a month

**Reply by mufeedcm** [reply]
did u get it 😄

**Comment by Ntshangase** [comment]
Did you try using a proof of registration document(convert it to the right format requested) instead of the student card, I think it will be easier that way. Try uploading  documents instead of taking the picture using the computer camera, It think that will provide better quality documents. 

That's if you already haven't tried this.

**Reply by Sidharth-Malik0** [reply]
What documents to be specific?

**Reply by Ntshangase** [reply]
Proof of academic status, to show you are a registered student at that institution. It just has to be dated, include name of institution and your name.

This is where I'm getting this information: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Reply by Ntshangase** [reply]
I just noticed something in your student ID:  Did you notice how it says:  " Parent ID card 2022-2023 ", It seems to me that you made this Student ID last year hence the " 2022 -2023" and you need something which says "2023 - 2024" for example because that card only tells me you were registered last year not this year and for GitHub you need to be a currently enrolled student.

**Comment by SocialBoothlv** [comment]
GitHub is an exceptional platform for developers and software enthusiasts, enabling them to host, review, and manage codebases, among other things. However, uploading projects to GitHub can be a challenge for some users, especially for those who are new to the platform.

One of the most common issues that users face when uploading projects to GitHub is getting their code to appear correctly. GitHub uses a specific markup language called Markdown, and it can take some time to get used to. Additionally, users may encounter conflicts when trying to merge changes from multiple contributors, leading to errors and confusion.

To overcome these issues, it's important to have a good understanding of Markdown and version control systems, such as Git. Additionally, it's crucial to regularly test the code and ensure that all changes are documented, which can help minimize conflicts and errors.

Overall, while uploading projects to GitHub can be a challenge, it's well worth the effort for the benefits it offers in terms of collaboration, community, and project management. By following best practices and taking the time to learn the platform, developers can leverage the power of GitHub to its fullest potential.
Regards **[SOCIAL BOOTH LV](https://www.socialboothlv.com/)**

**Comment by jarvisa98** [comment]
If you're facing difficulties in accessing the benefits of the GitHub Student Developer Pack, there are a few steps you can take to resolve the issue solve the [rope hero mod apk ]([https://ropeheroapk.pro/](https://ropeheroapk.pro/rope-hero-vice-town-1-3-1-mod-apk/))

Check eligibility: Ensure that you meet the eligibility criteria for the GitHub Student Developer Pack. Generally, it is available to students who are currently enrolled in a degree or diploma-granting course at an accredited institution.

Verify your student status: GitHub requires you to verify your student status to gain access to the benefits. Make sure you have completed the verification process by submitting the required documentation, such as a student ID card, enrollment letter, or transcript.

**Comment by jarvisa98** [comment]
The GitHub Student Developer Pack is an incredible resource for students, offering access to premium developer tools for free. From cloud services to design software and coding platforms to the [nica lotto times](https://tiemposnicaloto.com/)
